### PR TITLE
chore(CX-3624): Prepare feature flags for the release

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-3624] 

### Description
Turn off “Do more on Artsy” feature flag
Turn on “Meet you new art advisor” feature flag

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3624]: https://artsyproduct.atlassian.net/browse/CX-3624?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/store/config/features.ts [hunk 1]
```diff
@@ -256,13 +256,13 @@ export const features = defineFeatures({
   },
   AREnableDoMoreOnArtsyRail: {
     description: "Enable Do More on Artsy Rail",
-    readyForRelease: true,
+    readyForRelease: false,
```
Comment (MounirDhahri) [inline src/app/store/config/features.ts:259]: we can keep this as true as well I believe in case we want to revert this

**Review by MounirDhahri — APPROVED** [review]
thanks!

File: src/app/store/config/features.ts (hunk 1 above)
Comment (dariakoko) [reply src/app/store/config/features.ts:259]: oki, done

Comment (MounirDhahri) [reply src/app/store/config/features.ts]: great

**Pull request merged**